CDP4Reporting/DataSource/ReportingDataSourceNode.cs

```diff
@@ -257,7 +254,10 @@ private T GetRowRepresentation()
         /// <param name="table"></param>
         internal void AddDataRows(DataTable table)
         {
-            table.Rows.Add(this.GetDataRow(table));
+            if (this.IsVisible && this.filterCategory.Child == null)
```

Comment:
Yes. Things changed here. I only want the "deepest" nested category values in the result tree.
Otherwise the result contains hierarchical data and that is not the way to go for tabular data in a Report Definition.